docs/usage/self-hosted-configuration.md

```diff
@@ -136,6 +136,10 @@ To create the key pair with openssl use the following commands:
 
 Override this object if you wish to change the URLs that Renovate links to, e.g. if you have an internal forum for asking for help.
 
+## redisUrl
+
+If this value is set then Renovate will use Renovate for its global cache instead of the local file system. The global cache is used to store lookup results (e.g. dependency versions and release notes) between repositories and runs. Example url: `redis://localhost`.
```

Comment:
```suggestion
If this value is set then Renovate will use Redis for its global cache instead of the local file system. The global cache is used to store lookup results (e.g. dependency versions and release notes) between repositories and runs. Example url: `redis://localhost`.
```